src/NLog.StructuredLogging.Json.Tests/NLog.StructuredLogging.Json.Tests.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\..\NLog.StructuredLogging.Json.Common.props" />
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
```

Comment:
`dotnet test` works on the dll when run in `\NLog.StructuredLogging.Json\src\NLog.StructuredLogging.Json.Tests` 
If I make it an exe, it will give an error that it needs a `Main` method.